001-cron.php

```diff
@@ -14,6 +14,11 @@
  * @return bool
  */
 function wpcom_vip_use_core_cron() {
+	if ( VIP_GO_BASIC_AUTH ) {
```

Comment:
To avoid needing to rebuild all wp-configs straight away, can this include a `defined()`? Otherwise we're headed for notice-palooza.